test/conformance/service_test.go

```diff
@@ -315,15 +315,15 @@ func TestReleaseService(t *testing.T) {
 		expectedThirdRev  = helloWorldText
 	)
 
-	objects, err := test.CreateRunLatestServiceReady(logger, clients, &names, &test.Options{})
+	objects, err := test.CreateReleaseServiceWithLatest(logger, clients, &names, &test.Options{})
 	if err != nil {
 		t.Fatalf("Failed to create initial Service %v: %v", names.Service, err)
 	}
-	firstRevision := names.Revision
+	revisions := []string{names.Revision}
```

Comment:
I'd kind of like to see this deal with more interesting traffic shapes, e.g. `revisions := []string{names.Revision, "@latest"}` here and `50%` below.

I think my comment above is probably adequate for this PR, so feel free to follow up with the rest, if you want.

Reply:
I'd rather do this separately. Since I might want to change how the test works.